lib/snitcher/api/client.rb

```diff
@@ -227,7 +227,8 @@ def add_tags(token, tags=[])
   #
   # @return [Array<String>] list of the remaining tags on the Snitch.
   def remove_tag(token, tag)
-    delete("/v1/snitches/#{token}/tags/#{tag}")
+     path = "/v1/snitches/#{token}/tags/#{tag}"
+     delete(URI.encode(path))
```

Comment:
The indention is wrong here (3 spaces instead of 2). I'll get it fixed so we can get this merged.